CHANGELOG.md

```diff
@@ -21,6 +21,7 @@
 - Disable resource link span ingestion. ([#2647](https://github.com/getsentry/relay/pull/2647))
 - Collect `http.decoded_response_content_length`. ([#2638](https://github.com/getsentry/relay/pull/2638))
 - Add TTID and TTFD tags to mobile spans. ([#2662](https://github.com/getsentry/relay/pull/2662))
+- Add support for `request.url` in the `Event` `Getter`. ([#2675](https://github.com/getsentry/relay/pull/2675))
```

Comment:
There is already a changelog entry above for this change, titled:
> Allow access to more context fields in dynamic sampling and metric extraction.

Could you amend the changelog entry there? That entry is written in a more user-facing way.

Reply:
Sure!